dash/development/base_component.py

```diff
@@ -155,16 +152,13 @@ def __init__(self, **kwargs):
                         id_suffix,
                     )
                 else:
-                    # Otherwise import the package and extract the version number
                     error_string_prefix = error_string_prefix.format(
```

Comment:
```suggestion
                    # Otherwise import the package and extract the version number
                    error_string_prefix = error_string_prefix.format(
```